sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonScanner.java

```diff
@@ -120,6 +120,7 @@ private void scanFile(InputFile inputFile) {
     SubscriptionVisitor.analyze(checksBasedOnTree, visitorContext);
     saveIssues(inputFile, visitorContext.getIssues());
 
+    visitorContext.rootTree().accept(new SymbolVisitor(context.newSymbolTable().onFile(inputFile)));
```

Comment:
tiny detail but I personally think it would be more readable with : 

`new SymbolVisitor(context.newSymbolTable().onFile(inputFile)).visitFileInput(visitorContext.rootTree());` 

feel free to discard the suggestion.